### DISCUSSION TITLE
is there functional overlap between litee and plenary?

### DISCUSSION THREAD

**Discussion by molleweide** [ldelossa/litee.nvim#98] [General]
Hi again Louis,

May I ask what is your thoughts on the difference between litee and plenary?

**Comment by ldelossa** [comment]
The major reason litee.nvim exists is to create a reusable panel and tree module. The Panel acts like a vscode panel where multiple plugins can register and be displayed in a stack. 

Other functionality was built into litee.nvim organically as the plugins I were writing needed specific reusable bits.
